test/unit/kubernetes-deploy/ejson_secret_provisioner_test.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+require 'test_helper'
+
+class EjsonSecretProvisionerTest < KubernetesDeploy::TestCase
+  include FixtureDeployHelper
```

Comment:
I thought `FixtureDeployHelper` was intended for integration tests that require deploying resources. It's a bit sketchy that the unit test will have access to helpers like `deploy_dir`.
Maybe we could move `fixture_path` that you're using in this test into `KubernetesDeploy::TestCase`?